### PR TITLE
Feat/deliverable2

### PR DESCRIPTION
**Pull request by gerardo-santos**
## Description
```
In this PR the filter by category functionality was added in the product list page using mock data.

- Products were rendered in the product list page.
- The products can be filtered by category
- The UI pagination is shown
- A loading spinner appears when entering the product list page.

```
## Type of Change

```
Delete options that are not relevant.
```

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?
```
Add a brief description of how can be tested this deliverable step by step. 

- Making sure category filters work

```
## Any background context you want to provide?
```
deliverable 2 of capstone project 4
```
## What is the deliverable?
```
deliverable 2 of capstone project 4
```
## Screenshots and links

https://condescending-blackwell-f98c77.netlify.app/

image

image

image

image

image

image

## Notes
```
Only mock data has been used.
```
## Checklist
N/A

### CODE REVIEW

**Comment by danbenitezf** [comment]
OMG, I thought I already reviewed this. Please if I don't review your PRs feel free to remind me on slack

File: src/components/ProductCard.jsx [hunk 1]
```diff
@@ -1,14 +1,28 @@
-import PropTypes from 'prop-types';
 import { StyledCard } from './styles/Card.styled';
 import { CardImage } from './styles/CardImage.styled';
 import { CardTitle } from './styles/CardTitle.styled';
+import { ProductCardDetails } from './styles/ProductCardDetails.styled';
 import Button from 'react-bootstrap/Button';
+import PropTypes from 'prop-types';
```
Comment (danbenitezf) [inline src/components/ProductCard.jsx:6]: Just a reminder for the import order.

**Review by danbenitezf — COMMENTED** [review]
Everything seems to be good. Some things may need to be changed for the API but as I was so late to review this 🐢 and you already finished deliverable 3, you may as well have changed or thought of those changes.

**Comment by gerardo-santos** [comment]
> OMG, I thought I already reviewed this. Please if I don't review your PRs feel free to remind me on slack

Don't worry man, I understand you are currently working in many tasks and projects. Thanks for the advice, and I will make sure the imports order is fixed in future deliveries.

**Pull request merged**